analysis/dataflow/dataflow_test.go

```diff
@@ -522,55 +525,47 @@ func BenchmarkReaching(b *testing.B) {
     return a    //11
     //END
   }
-    `
-	var config loader.Config
-	f, err := config.ParseFile("testing", src)
-	if err != nil {
-		b.Error(err.Error())
-		b.FailNow()
-	}
-
-	config.CreateFromFiles("testing", f)
-
-	prog, err := config.Load()
-
-	if err != nil {
-		b.Error(err.Error())
-		b.FailNow()
-	}
+    `)
 
 	// create CFG and perform analyses
 	for n := 0; n < b.N; n++ {
-		cfg := cfg.FromFunc(f.Decls[0].(*ast.FuncDecl))
-		DefUse(cfg, prog.Created[0])
-		LiveVars(cfg, prog.Created[0])
+		cfg := cfg.FromFunc(c.prog.Syntax[0].Decls[0].(*ast.FuncDecl))
```

Comment:
note: everything that does c.prog.Syntax[0] is basically incorrect, many files are loaded, even if this is only used in benchmarking (I think?) so maybe doesn't matter